modules/luci-mod-dashboard/luasrc/view/dashboard/index.htm

```diff
@@ -0,0 +1,371 @@
+<%#
+ Copyright 2019-2020 ZHANG Zhao <Zhao.Zhang2@etu.univ-grenoble-alpes.fr>
+ Licensed to the public under the Apache License 2.0.
+-%>
+
+<% css = [[
+
+/**
+ * Bootstraps Basics Styles
+**/
+
+.label {
+	display: inline;
+	padding: .2em .6em .3em;
+	font-size: 75%;
+	font-weight: 700;
+	line-height: 1;
+	color: #fff;
+	text-align: center;
+	white-space: nowrap;
+	vertical-align: baseline;
+	border-radius: .25em;
+	font-family: "Helvetica Neue",Helvetica,Arial,sans-serif;
```

Comment:
Text inside the label looks badly positioned.
![image](https://user-images.githubusercontent.com/16000931/88321417-d94e6b80-ccba-11ea-9b91-52af681b883b.png)

Reply:
Thank you for your feedback. please tell me the name and version of the browser you are using?

Reply:
Firefox 78

Reply:
Okay thanks :)